pkg/console/operator/sync_v400.go

```diff
@@ -171,9 +171,9 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 }
 
 func SyncConsoleConfig(co *consoleOperator, consoleConfig *configv1.Console, route *routev1.Route) (*configv1.Console, error) {
-	host, err := routesub.GetCanonicalHost(route)
-	if err != nil {
-		return nil, err
+	host := routesub.GetCanonicalHost(route)
+	if host == "" {
+		return nil, errors.New("waiting on host")
```

Comment:
These errors will be transformed into custom `SyncError` in my next PR that will be used to report `Progressing` vs `Failing`.